server.go

```diff
@@ -54,7 +54,7 @@ type Server interface {
 func NewServer(address string, topics []string, config *ServerConfig) (Server, error) {
 	config = processServerConfig(config)
 
-	listener, err := quic.ListenAddr(address, config.TLSConfig, nil)
+	listener, err := quic.ListenAddr(address, config.TLSConfig, nil) //nolint:typecheck
```

Comment:
Ignoring `typecheck`? Can you please share the error here?

Reply:
It is a mistake. I will fix it.